pkg/performanceprofile/utils/testing/testing.go

```diff
@@ -19,9 +19,10 @@ const (
 	// ReservedCPUs defines the reserved CPU set used for tests
 	ReservedCPUs = performancev2.CPUSet("0-3")
 	// OfflinedCPUs defines the Offline CPU set used for tests
-	OfflinedCPUs     = performancev2.CPUSet("6-7") // SingleNUMAPolicy defines the topologyManager policy used for tests
-	SingleNUMAPolicy = "single-numa-node"
+	OfflinedCPUs = performancev2.CPUSet("6-7")
```

Comment:
Though these values that we add to ReservedCPUs , OfflinedCPUs, IsolatedCPUs are fine, But when running these tests in Baremetal, we can't use these values and we will have to fill them with what Baremetal systems have.  Because if i run them as is, tuned will start complaining if we don't account for the rest of the cpus

Reply:
Yes sorry, it was for local testing purposes, fixed.